README.md

```diff
@@ -0,0 +1,149 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 'Hello Microverse' Project <a name="about-project"></a>
+
+Hello Microverse is a project for the Microverse Students to make sure they are fully understand with the professional Github pull and work flow and understanding about linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- Make understand work flow of GitHub
+- Make Understand Linters
+- Make Understand about the professional template writing.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+  Installation of Git
+
+  Installation of NPM
+
+  A working browser such as chomre or firefox
+
+  Internet connection
+
+  A code editor like Visual Studio Code
+
+  Installation of Live Server plugging in Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+ 
+  cd to the desire folder
+  
+  paste
+```sh
+  git clone https://github.com/popey17/Hello-World.git
+```
+
+### Install
+
+Install this project with:
+
+  follow the instruction from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) link to install Linters for html-css.  
+
+### Usage
+
+To run the project, execute the following command:
+
+  run the HTML file form the live server via VSCode or directly open HTML file.
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@popey17](https://github.com/popey17)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aung-myat-kyaw-3214a8220/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add some details and nore lessons to understand about GitHub professional usages.
```

Comment:
- [ ] Please add up the remaining two as bullet points. just as its stated on the template given. 

## View:
![image](https://user-images.githubusercontent.com/50990925/218930307-26811964-8eb9-472c-bf11-015c40b79012.png)